contrib/flake.nix

```diff
@@ -10,7 +10,14 @@
     {
       overlay = final: prev: {
 
```

Comment:
what about just `clangStdenv = prev.clang14Stdenv;` ? would be easier on the eyes if that worked

Reply:
I was only able to get something like this to work:

```
        neovim = let
          clangStdenv = prev.clang14Stdenv;

          stdenv = if final.stdenv.isDarwin then
            clangStdenv
          else
            final.stdenv;
        in (final.neovim-unwrapped.override {
          inherit stdenv;
# ...
```

Which is almost certainly not what you had in mind?

I don't know if there is a way to override the `clangStdenv` (in a nice way) in an overlay ...

The current version is based on how the `stdenv` is overriden for `neovim-debug`.

Reply:
I was thinking of doing 
```
overlay = final: prev: {
     clangStdenv = prev.clang14Stdenv; 
    # leave the rest unchanged
    neovim = ... 
}}}
``` 
I've asked on matrix why we kept clang11 as a default and a bump is likely but after the nixos 23.05 release so maybe this isn't worth pursuing after all, as overriding clang  makes the flake diverge more from nixpkgs ?
sorry to have made you lose your time but thanks to you the internet now knows how to do this.